From 97c7af91b8b1c1c0b01e3a7ae4478c1413bd768c Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Fri, 7 Jun 2019 11:16:49 +0000
Subject: ozone/wayland: Fix method prototype match

CreateShmBasedBuffer uses uint64_t type for the length parameter,
but the definition uses size_t instead.

This CL fixes this inconsistency.

Bug: 578890
Change-Id: I3deac38340422d7c2098d226966312ad16eb831b
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/1649156
Reviewed-by: Antonio Gomes <tonikitoo@igalia.com>
Commit-Queue: Maksim Sisov <msisov@igalia.com>
Cr-Commit-Position: refs/heads/master@{#667080}

diff --git a/ui/ozone/platform/wayland/host/wayland_buffer_manager_host.cc b/ui/ozone/platform/wayland/host/wayland_buffer_manager_host.cc
index b6c85ef70fb4..3f163a012c31 100644
--- a/ui/ozone/platform/wayland/host/wayland_buffer_manager_host.cc
+++ b/ui/ozone/platform/wayland/host/wayland_buffer_manager_host.cc
@@ -558,7 +558,7 @@ void WaylandBufferManagerHost::CreateDmabufBasedBuffer(
 void WaylandBufferManagerHost::CreateShmBasedBuffer(
     gfx::AcceleratedWidget widget,
     mojo::ScopedHandle shm_fd,
-    size_t length,
+    uint64_t length,
     const gfx::Size& size,
     uint32_t buffer_id) {
   DCHECK(base::MessageLoopCurrentForUI::IsSet());
-- 
2.20.1

